### PR TITLE
✨ Resolve ATTRIBUTION_REPORTING_STATUS in attributionsrc/src for generated pixels

### PR DESCRIPTION
**Pull request by emensc52**
Replace ATTRIBUTION_REPORTING_STATUS in the url to set the parameter to an enum based on attributionsrc existence and whether attribution reporting is supported.

### CODE REVIEW

File: src/pixel.js [hunk 1]
```diff
@@ -67,10 +70,25 @@ function createImagePixel(win, src, noReferrer = false, attributionSrc) {
...
-  if (isAttributionReportingAllowed(win.document)) {
-    image.attributionsrc = attributionSrc;
+
+  let attributionReportingStatus =
+    AttributionReportingStatus.ATTRIBUTION_DATA_UNSPECIFIED;
+  if (attributionSrc != null) {
+    if (isAttributionReportingAllowed(win.document)) {
+      attributionReportingStatus =
+        AttributionReportingStatus.ATTRIBUTION_DATA_PRESENT_AND_POLICY_ENABLED;
+      attributionSrc = attributionSrc.replace(
```
Comment (powerivq) [inline src/pixel.js:80]: Need to use urlReplacementService to do argument substitution: https://github.com/ampproject/amphtml/blob/3c58857f178d2b81a62304d61ab0419cf00ba569/extensions/amp-ad-exit/0.1/amp-ad-exit.js#L208

Using `replace` would not be safe.

**Review by powerivq — CHANGES_REQUESTED** [review]

File: src/pixel.js (hunk 1 above)
Comment (emensc52) [reply src/pixel.js]: Updated, thanks.

File: src/utils/privacy-sandbox-utils.js [hunk 2]
```diff
@@ -15,3 +15,14 @@ export function isAttributionReportingAvailable(doc) {
 export function isAttributionReportingAllowed(doc) {
   return doc.featurePolicy?.allowedFeatures().includes('attribution-reporting');
 }
+
+/**
+ * Indicates the status of the `attribution-reporting` API.
+ * @enum
+ */
+export const AttributionReportingStatus = {
```
Comment (powerivq) [inline src/utils/privacy-sandbox-utils.js:23]: Can you change the amp-ad-exit to reuse this definition?

**Review by powerivq — CHANGES_REQUESTED** [review]
Otherwise lgtm

**Review by powerivq — APPROVED** [review]

**Review by erwinmombay — APPROVED** [review]
approving. but will let @powerivq give final approval and merge

**Comment by powerivq** [comment]
Can you rebase against latest main branch?

**Pull request merged**